repeat_number.cpp

```diff
@@ -0,0 +1,10 @@
+#include<stdio.h>
+int main()
+{
+	int i;
+		for(i=4;i<=8;++i)
+		{
+			printf("%d",4);
```

Comment:
why using `%d` here?
You are printing constant value (constant values are also called literals).

use 
```printf("4");```
instead of
```printf("%d",4);```